cli/types/cypress.d.ts

```diff
@@ -2633,6 +2633,13 @@ declare namespace Cypress {
     cmdKey: boolean
   }
 
+  interface CookieOptions extends Loggable, Timeoutable {
+    /**
+     * Domain to set cookies on or get cookies from
```

Comment:
```suggestion
     * Domain to set cookies on or get cookies from. When domain is not provided, the current domain is used.
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/24471/commits/73bb4c36d60c0418316b06107ef21f2dd3f1804a